src/RoleObjects/MenuData.js

```diff
@@ -89,6 +89,7 @@ export default class MenuData extends SelectData {
 
       // this doesn't handle menus starting/ending with a separator
       if (this._children[nextIndex].accessible.role === ROLES.SEPARATOR) {
+        // eslint-disable-next-line @typescript-eslint/no-unused-expressions
         evt.keyCode === KeyCodes.up ? nextIndex-- : nextIndex++;
```

Comment:
I think rather than marking for this specific line, we should add a property `"allowTernary": true` in .eslintrc,js. As we tend to use ternary most of the times. https://eslint.org/docs/latest/rules/no-unused-expressions#allowternary